gr-blocks/lib/plateau_detector_fb_impl.h

```diff
@@ -44,6 +44,9 @@ class plateau_detector_fb_impl : public plateau_detector_fb
                       gr_vector_int &ninput_items,
                       gr_vector_const_void_star &input_items,
                       gr_vector_void_star &output_items);
+
+  virtual void set_threshold(float threshold);
```

Comment:
`const float`

Reply:
I can certainly change these to "const float" ... but I don't see why.

The "const" here just indicates that the "threshold" variable cannot be modified, right? It's a compiler directive & likely won't change the resulting assembly code, and hence I greatly doubt it'll change the runtime performance. I'm pretty sure this variable is not modified in the method implementation, and it's simple enough to change the code to not modify it (just copy to a non-const variable). Even without the const, it doesn't matter if the variable is changed, since it's value is not returned to the caller.

Hence, I don't see a good reason to add the "const" .... unless this is a GREP thing ... ? I haven't had time to look through the coding GREP yet.